pipeline/load/display_metrics.py

```diff
@@ -1,59 +1,34 @@
-import operator
+from typing import Dict, List
 
 import prefect
 from prefect import task
 
-from pipeline.types import MatchingMetrics, MatchingOutput, User
+from pipeline.types import MatchingMetrics, MatchingOutput, User, UserId
+
+
+def render_counts_per_user(
```

Comment:
I created a helper function for three reasons:
1. Easier to test because it returns a list of strings
2. Easier to reuse for future user count metrics because it is not specific to proposed matches per user
3. Easier to read because it separates the names and sorting from the logging code

Think about how to break code down into smaller functions for testing, reuse, and readability. But don't break them down so small that it becomes tedious to jump around and understand what the whole code does.